tests/topotests/bgp-evpn-vxlan_topo1/host2/ospfd.conf

```diff
@@ -0,0 +1 @@
+!
```

Comment:
could you fix all those github warnings? (missing \n)

![Screenshot from 2020-03-30 17-26-01](https://user-images.githubusercontent.com/3352707/77923771-900d8280-72ab-11ea-926f-52cf9b0047a9.png)